components/bal/language-preview.js

```diff
@@ -12,17 +14,34 @@ function LanguagePreview({nomAlt}) {
       <Pane fontStyle='italic' fontSize={14} fontWeight='lighter'>
         <Tooltip
           content={
-            <UnorderedList>
+            <UnorderedList display='flex' flexDirection='column' padding={0} margin={0}>
               {Object.keys(nomAlt).map(language => (
-                <ListItem color='white' key={language}>
+                <ListItem
+                  key={language}
+                  color='white'
+                  listStyleType='hidden'
+                  display='grid'
+                  alignItems='start'
+                  gridTemplateColumns='22px 1fr'
+                  gap={8}
+                  marginLeft={-15}
+                >
+                  <Image
+                    src={isFlagExist ? `/static/images/flags/${language}.svg` : '/images/icons/flags/ntr.svg'}
+                    height={22}
+                    width={22}
+                    alt={languesRegionales.some(lr => lr.code === language) ? (
+                      `Nom de la voie en ${languesRegionales.find(lr => lr.code === language).label}`
+                    ) : (
+                      'Nom de la langue régionale introuvable'
+                    )}
```

Comment:
Le double calcule `some` + `find` est un peu overkill.

Le plus simple serait de d'utiliser `find` en amont et faire :
```suggestion
                    alt={language ? (
                      `Nom de la voie en ${language.label}`
                    ) : (
                      'Nom de la langue régionale non supportée'
                    )}
```